data_structures/stacks/evaluate_postfix_notations.py

```diff
@@ -0,0 +1,48 @@
+# The Reverse Polish Nation also known as Polish postfix notation or simply postfix notation
+# https://en.wikipedia.org/wiki/Reverse_Polish_notation
+# Classic examples of simple stack implementations
+# Valid operators are +, -, *, /. Each operand may be an integer or another expression.
```

Comment:
This is called a docstring which explains the intent of this file.  The docstring is used by [pydoc](https://docs.python.org/3/library/pydoc.html) if you run the command `python3 -m pydoc data_structures/stacks/evaluate_postfix_notations.py`

By converting this to a comment, this PR would remove that documentation.